node/src/env0-deploy-flow.js

```diff
@@ -14,19 +18,22 @@ const assertRequiredOptions = (options) => {
 }
 
 const runDeployment = async (command, options, environmentVariables) => {
+  const existingOptions = persistentOptions.read();
+
+  options = { ...existingOptions, ...options, }
```

Comment:
I think merging CLI args with other configuration mediums, should be done in the 'config-manager'

Reply:
1f04b63